core/src/main/java/org/infinispan/commands/statetransfer/ConflictResolutionCancelCommand.java

```diff
@@ -0,0 +1,50 @@
+package org.infinispan.commands.statetransfer;
+
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.commons.util.IntSet;
+import org.infinispan.factories.ComponentRegistry;
+import org.infinispan.remoting.transport.Address;
+import org.infinispan.statetransfer.StateProvider;
+import org.infinispan.util.ByteString;
+import org.infinispan.util.concurrent.CompletableFutures;
+
+/**
+ * Cancel conflict resolution.
+ *
+ * @author Ryan Emerson
+ * @since 11.0
+ */
+public class ConflictResolutionCancelCommand extends AbstractStateTransferCommand {
```

Comment:
can be removed and use `StateTransferCancelCommand`
I'm aware `boolean isReturnValueExpected()` is different but both commands are sent async (any response is discarded) and both commands return `null`.